### PR TITLE
Add C# and VB.NET to quickstart

### PR DESCRIPTION
**Pull request by nbarbettini**

### CODE REVIEW

File: source/003_quickstart.rst [hunk 1]
```diff
@@ -4,9 +4,21 @@
 3. Quickstart
 *************
 
-This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath REST API.
+.. only:: not (csharp or vbnet)
```
Comment (nbarbettini) [inline source/003_quickstart.rst]: This needs to be language specific, since only REST will deal with cURL

File: source/003_quickstart.rst [hunk 2]
```diff
@@ -4,9 +4,21 @@
...
 
-These instructions assume that you have `cURL <http://curl.haxx.se/download.html>`_ installed on your system and that you have already completed the steps in the :ref:`Set-up chapter<set-up>`, and now have:
+    This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath REST API.
+
+.. only:: csharp or vbnet
+
+    This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath .NET SDK.
+
+.. only:: not (csharp or vbnet)
```
Comment (nbarbettini) [inline source/003_quickstart.rst]: Temporary, this could probably be `only:: rest`

File: source/003_quickstart.rst [hunk 3]
```diff
@@ -35,14 +59,38 @@ Let's get started!
 
 Before you can create user Accounts you'll need to retrieve your Stormpath Application. An Application in Stormpath represents the project that you are working on. This means that, if you're building a web app named "Lightsabers Galore", you'd want to name your Stormpath Application "Lightsabers Galore" as well. By default, your Stormpath Tenant will have an Application already created for you to use. We will use this Application, named "My Application", for the quickstart.
 
-In our examples below we will use the mock API Key from the :ref:`First Time Set-Up <set-up>` chapter. You should replace this mock Key with your own, valid key::
+.. only:: not (csharp or vbnet)
```
Comment (nbarbettini) [inline source/003_quickstart.rst]: This is REST-specific, or at least doesn't apply to me :smile:

File: source/003_quickstart.rst [hunk 4]
```diff
@@ -294,14 +379,18 @@ Now we have a user Account that can use your Application. But how do you authent
 
 .. only:: nodejs
 
-  .. literalinclude:: code/nodejs/quickstart/authentication_attempt 
+  .. literalinclude:: code/nodejs/quickstart/authentication_attempt
       :language: javascript
 
-If the authentication attempt fails, you will see an error response instead:
```
Comment (nbarbettini) [inline source/003_quickstart.rst]: This needs to be language-specific too, I believe. @dogeared or @rdegges, does this apply to your respective languages, or would you have to rewrite it (like me) to talk about handled exceptions?

**Comment by nbarbettini** [comment]
There's some extra noise in the commits, sorry. Looks like my editor decided to reformat some line breaks or something.

**Comment by nbarbettini** [comment]
@jakubswiatczak No longer WIP, other than the discussion points above.

File: source/003_quickstart.rst (hunk 2 above)
Comment (rdegges) [inline source/003_quickstart.rst]: This is fine =)

File: source/003_quickstart.rst (hunk 3 above)
Comment (rdegges) [inline source/003_quickstart.rst]: +1

File: source/003_quickstart.rst (hunk 4 above)
Comment (rdegges) [inline source/003_quickstart.rst]: Correct. Could you add in the language specific stuff for yourself?

**Pull request merged**